schemas/test/en/test_progress_value_source_calculated_summary_extended.json

```diff
@@ -541,21 +541,38 @@
                                     "!=": [
                                         {
                                             "source": "progress",
-                                            "selector": "block",
-                                            "identifier": "calculated-summary-block"
+                                            "selector": "section",
+                                            "identifier": "section-1"
                                         },
                                         "COMPLETED"
                                     ]
                                 }
                             }
+                        },
+                        {
```

Comment:
Not sure if this is a schema thing or a runner thing - when I answer the non-skippable question, before the skippable one is enabled by completing section 1, I can edit it via "View Answers" as expected. However if I then enable the skippable question by completing section 1 and answer it, I am only given the option to edit the skippable question and not the non-skippable one. This latter behaviour also occurs if I complete section 1 before ever trying section 4. 

https://user-images.githubusercontent.com/108286538/234303838-7ba8da2e-048b-41c8-b704-98833028ae2e.mov

Reply:
I think this is down to the schema again, you would only see the other question again if there was a section summary